xml/docu_styleguide.manage.xml

```diff
@@ -153,14 +268,14 @@
     Each header of a &suse; XML file includes the entity declaration file (by
     means of an entity):
    </para>
-<screen>&lt;!ENTITY % entities SYSTEM "entity-decl.ent"&gt;
+<screen>&lt;!ENTITY % entities SYSTEM "generic-entities.ent"&gt;
 %entities;</screen>
    <example xml:id="ex-entity">
-    <title>Excerpt from an <filename>entity-decl.ent</filename></title>
-<screen>&lt;!ENTITY<co xml:id="co-declaration"/> server<co xml:id="co-definition"/> "sun<co xml:id="co-value"/>"&gt;
-&lt;!ENTITY domain  "example.org"&gt;
-&lt;!ENTITY host    "&amp;server;.&amp;domain;<co xml:id="co-entity"/>"&gt;
-&lt;!ENTITY hosttag "&lt;uri xmlns='http://docbook.org/ns/docbook'&gt;&amp;host;&lt;/uri&gt;<co xml:id="co-tag"/>"&gt;</screen>
+    <title>Excerpt from <filename>product-entities.ent</filename></title>
+<screen>&lt;!ENTITY<co xml:id="co-declaration"/>product-sp<co xml:id="co-definition"/> "Code stream<co xml:id="co-value"/>"&gt;
+&lt;!ENTITY product-ga  "15"&gt;
+&lt;!ENTITY productnumber-regurl    "&amp;product-ga;.&amp;product-sp;<co xml:id="co-entity"/>"&gt;
+&lt;!ENTITY productnumber "&lt;phrase xmlns='http://docbook.org/ns/docbook' role='productnumber'&gt;&lt;/phrase"&gt;<co xml:id="co-tag"/>"&gt;</screen>
```

Comment:
```suggestion
&lt;!ENTITY productnumber "&lt;phrase xmlns='http://docbook.org/ns/docbook' role='productnumber'&gt;15&lt;/phrase"&gt;<co xml:id="co-tag"/>"&gt;</screen>
```

Reply:
An entity value was still missing, I added `15` as a concrete example for a code stream number here.